src/block/horizontal-scroller/save.js

```diff
@@ -54,7 +54,10 @@ export const Save = props => {
 		 >
 			 { attributes.generatedCss && <style>{ attributes.generatedCss }</style> }
 			 <CustomCSS.Content attributes={ attributes } />
-			<div className={ contentClassNames }>
+			<div
```

Comment:
Check my suggestion for the progress circle, this prop would need to be added in by default, and then the deprecation filter would remove it. This is so that our save.js always contains our latest version, and the filters are there to modify it for older versions